package-exclude.json

```diff
@@ -30,8 +30,7 @@
         "redux-mock-store",
         "surge",
         "webpack-bundle-analyzer",
-        "webpack-dev-server",
-        "webpack-dev-server-without-h2",
+        "webpack-stats-plugin",
```

Comment:
The webpack config has this line:
```js
var StatsWriterPlugin = require('webpack-stats-plugin').StatsWriterPlugin;
```

That makes me think webpack will crash/fail if the plugin isn't present, but we do need to run webpack in a packaged environment meaning we can't skip it. I didn't test this though and perhaps JS will happily proceed if the package can't be loaded, but it'd surprise me.